plasmapy/physics/tests/test_quantum.py

```diff
@@ -1,9 +1,21 @@
-import numpy as np
-import pytest
-import astropy.units as u
-from ...constants import c, h
-from ..quantum import deBroglie_wavelength
-
+if __name__ == "__main__":    
```

Comment:
As before, this looks weird-ish to me. It seems to me like with properly written and properly used code you shouldn't need to check for this... I could really use a second opinion on this. @PlasmaPy/plasmapy-community?

Reply:
This [link](https://stackoverflow.com/questions/419163/what-does-if-name-main-do#419185) may explain it better.

Reply:
Sure, I'm aware of the practice of doing that! However, what I'm trying to figure out is what's the optimal way of achieving the ability to execute a script visibly testing some library functions while debugging it, while also not having to deal with this sort of visibility mess.

Here's a few cases of this that I've found:
* https://github.com/matplotlib/matplotlib/blob/e6ef481a32ed3ec982ce4dabdc627334fb1db2e4/lib/mpl_toolkits/axisartist/clip_path.py#L140
* https://github.com/matplotlib/matplotlib/blob/7131876063f933bc5e5c5d768722dd81bb69038e/lib/mpl_toolkits/axisartist/angle_helper.py#L439
* https://github.com/numpy/numpy/blob/c90d7c94fd2077d0beca48fa89a423da2b0bb663/numpy/lib/tests/test_arrayterator.py#L50

So while I completely agree it's nice to be able to run `python whatevermodulewereworkingon.py` and see how it's coming along while development, we should probably not need to do this kind of visibility double-checking to achieve that.